src/scripts/main.js

```diff
@@ -2,7 +2,27 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
-
 // write your code here
+const table = document.getElementsByClassName('dashboard')[0];
+const getCentury = (deathAge) => {
+  return Math.ceil(deathAge / 100);
+};
+
+const getFullGender = (sex) => {
+  return sex === 'f' ? 'Female' : 'Male';
+};
+
+const getRow = (person) => {
+  return `<tr>
+    <td> ${person.name} </td>
+    <td>${getFullGender(person.sex)}</td>
+    <td>${person.born}</td>
+    <td>${person.died}</td>
+    <td>${person.died - person.born}</td>
+    <td>${getCentury(person.died)}</td>
+  </tr>`;
+};
+
+const tbody = table.children[0];
+
+tbody.innerHTML += people.map(person => getRow(person)).join('');
```

Comment:
```suggestion
tbody.innerHTML += people
  .map(getRow)
  .join('');
```
You can move every method on a new line. It will be more readable, I think.
And you can directly pass the callback function in `map` method if you have the separate one.

Reply:
Thanks, good points).